packages/testlab/README.md

```diff
@@ -2,10 +2,19 @@
 
 A collection of test utilities we use to write LoopBack tests.
 
-## `expect`
+## Overview
 
-[Should.js](https://shouldjs.github.io/) configured in "as-function" mode
-(global `Object.prototype` is left intact) with an extra chaining word `to`.
+Test utilities to help writing loopback-next tests:
+- Should.js: expect(actual, expected, [message]).
+- sinon: Standalone test spies, stubs and mocks, that Works with any unit testing framework.
+
```

Comment:
There are more utilities than just should + sinon: 

 - Helpers for creating `supertest` clients, most notably `createClientForApp` - see [client.ts](https://github.com/strongloop/loopback-next/blob/290fc7a60ac2ebe05826117f6788f99aab433d2f/packages/testlab/src/client.ts)
 - HTTP request/response mocks, see [shot.ts](https://github.com/strongloop/loopback-next/blob/290fc7a60ac2ebe05826117f6788f99aab433d2f/packages/testlab/src/shot.ts)